admin/src/main/resources/io/buoyant/admin/js/src/router_clients.js

```diff
@@ -22,7 +26,9 @@ define([
       return numClients < EXPAND_CLIENT_THRESHOLD;
     }
 
-    return function (metricsCollector, routers, $clientEl, routerName, clientTemplate, metricPartial, clientContainerTemplate, colors) {
+    return function (metricsCollector, routers, $clientEl, routerName, colors) {
+      var clientContainerTemplate = Handlebars.compile(routerClientContainerTemplate);
```

Comment:
These vars can now be hoisted out of the returned function... although this is only actually useful if we're instantiating these component more than once. (are we?)

Template compilation on the client is usually a very costly operation because of the string manipulation required.